editor/embeddable-autotester.js

```diff
@@ -324,6 +322,13 @@ function autotester(sceneText, nav, sceneName, extraLabels) {
   Scene.prototype.gosub = function scene_gosub(label, inChoice) {
     if (!seen[label.toLowerCase()]) this.oldGosub(label);
   }
+
+  // Params can't be filled with reasonable values (we don't know the types,
+  // let alone additional input constraints), so just treat as return.
+  // Randomtest will test gosubs with a good mix of parameters anyway.
+  Scene.prototype.params = function scene_params(data) {
```

Comment:
I don't think this is correct; I think we can just cut this override function and let quicktest do its thing and it will work.

The premise of quicktest is that it will fork copies of itself for all choice #options, and fork copies for every `*if` statement, exploring both true and false branches regardless of whether they're actually reachable. Generally speaking, as long as variables are of the right type, it _will_ have some semi-reasonable values to pass as parameters: whatever the values actually turned out to be during the quicktest run.

I tried some trivial tests on my machine; quicktest was able to run and successfully validate missing parameters in my tests. Did you find examples where quicktest failed incorrectly as a result of parameters?